sim/druid/ferocious_bite.go

```diff
@@ -0,0 +1,125 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+)
+
+type FerociousBiteRankInfo struct {
+	id           int32
+	level        int32
+	dmgBase      float64
+	dmgRange     float64
+	dmgPerCombo  float64
+	dmgPerEnergy float64
+}
+
+var ferociousBiteRanks = []FerociousBiteRankInfo{
+	{
+		id:           22568,
+		level:        32,
+		dmgBase:      14.0,
+		dmgRange:     16.0,
+		dmgPerCombo:  36.0,
+		dmgPerEnergy: 1.0,
+	},
+	{
+		id:           22827,
+		level:        40,
+		dmgBase:      20.0,
+		dmgRange:     24.0,
+		dmgPerCombo:  59.0,
+		dmgPerEnergy: 1.5,
+	},
+	{
+		id:           22828,
+		level:        48,
+		dmgBase:      30.0,
+		dmgRange:     40.0,
+		dmgPerCombo:  92.0,
+		dmgPerEnergy: 2.0,
+	},
+	{
+		id:           22829,
+		level:        56,
+		dmgBase:      45.0,
+		dmgRange:     50.0,
+		dmgPerCombo:  128.0,
+		dmgPerEnergy: 2.5,
+	},
+	// Rank 5 learned from the book.
+	// Implement this spell as having multiple ranks available later? Like e.g. Starfire?
+	// Or just uncomment this once the book is available?
+	// {
+	// 	id:           31018,
+	// 	level:        60,
+	// 	dmgBase:      52.0,
+	// 	dmgRange:     60.0,
+	// 	dmgPerCombo:  147.0,
+	// 	dmgPerEnergy: 2.7,
+	// },
```

Comment:
I think it should be ok to un-comment? The spells only get registered if the level is below the character's level

Reply:
Also PS I really like the formatting you used here

Reply:
No clue what they will do with SoD, but it's likely not available immediately on 60. Those books were made available with AQ, or DM, can't remember right now.